packages/recordio/README.md

```diff
@@ -0,0 +1,39 @@
+# RecordIO
+
+Package exports one function that parses RecordIO encoded records.
```

Comment:
How about: "This package provides a function to read records in the [RecordIO format](http://mesos.apache.org/documentation/latest/scheduler-http-api/#recordio-response-format) from the input string."